### ISSUE TITLE
Splitting the dataset into training set and test set

### ISSUE THREAD

**Issue by islandjoe** [islandjoe/hands-on-data-science#5]
Split dataset into 80% training set and 20% test set.
